### PR TITLE
docs(ClientOptions): change default value of sweepers in docs

### PR DESCRIPTION
**Pull request by Ryoh827**
**Please describe the changes this PR makes and why it should be merged:**

As described in #9579, the default value for `ClientOptions#sweepers` is `{threads: {interval: 3600, lifetime: 14400}}`, not `{}`.
So I have changed the documentation as follows

before
```
@property {SweeperOptions} [sweepers={}] Options for cache sweeping
```

After
```
@property {SweeperOptions} [sweepers=this.DefaultSweeperSettings] Options for cache sweeping
```

**Status and versioning classification:**

- This PR **only** includes non-code changes, like changes to documentation, README, etc.
- I know how to update typings and have done so, or typings don't need updating

### CODE REVIEW

**Comment by jaw0r3k** [comment]
Um worst way to do it ever
I would make a new typedef or smth but not like that

File: packages/discord.js/src/util/Options.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const { version } = require('../../package.json');
  * @property {IntentsResolvable} intents Intents to enable for this connection
  * @property {number} [waitGuildTimeout=15_000] Time in milliseconds that clients with the
  * {@link GatewayIntentBits.Guilds} gateway intent should wait for missing guilds to be received before being ready.
- * @property {SweeperOptions} [sweepers={}] Options for cache sweeping
+ * @property {SweeperOptions} [sweepers={threads: {interval: 3600, lifetime: 14400}}] Options for cache sweeping
```
Comment (jaw0r3k) [inline packages/discord.js/src/util/Options.js:38]:
```suggestion
 * @property {SweeperOptions} [sweepers=DefaultSweeperSettings] Options for cache sweeping=
```

the shortest way

Comment (Ryoh827) [reply packages/discord.js/src/util/Options.js:38]: Sorry.

Thank you for the suggestion.
I will modify it to your form.

Comment (jaw0r3k) [reply packages/discord.js/src/util/Options.js:38]: Btw you can just commit suggestion on github site

Comment (Ryoh827) [reply packages/discord.js/src/util/Options.js]: I've modified it by adding an additional 'this.' What do you think?

I used the following code and others as reference
https://github.com/discordjs/discord.js/blob/main/packages/discord.js/src/sharding/ShardingManager.js#L177

Sorry if this is wrong.

Comment (thehairy) [reply packages/discord.js/src/util/Options.js]: > the shortest way, but i am not too sure if its right

If that links properly to what it returns in the docs, then it's alright.
If not I think the original way is appropiate.

Comment (jaw0r3k) [reply packages/discord.js/src/util/Options.js]: Nah docs shows just `default` so lnks wony work there

Comment (jaw0r3k) [reply packages/discord.js/src/util/Options.js]: But this long syntax looks horrible in docs

**Review by Jiralite — APPROVED** [review]

**Review by kyranet — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**